src/main/java/InvalidTaskIndexException.java

```diff
@@ -0,0 +1,8 @@
+public class InvalidTaskIndexException extends DukeException {
+
+    InvalidTaskIndexException(String command) {
+        super("☹ OOPS!!! Please enter a valid task index to be " + (command.startsWith("done")
+                ? "marked as done."
+                : "deleted."));
```

Comment:
May I suggest grouping the ternary operator entirely in a new line as follows?

super("☹ OOPS!!! Please enter a valid task index to be " 
                + (command.startsWith("done") ? "marked as done." : "deleted."));